test/utils/unitUtils.js

```diff
@@ -0,0 +1,29 @@
+const { ethers } = require("ethers");
+const { BigNumber } = ethers;
+
+const ether = (amount) => {
+    const weiString = ethers.utils.parseEther(amount.toString());
+    return BigNumber.from(weiString);
+};
+
+const wei = (amount) => {
+    const weiString = ethers.utils.parseUnits(amount.toString(), 0);
+    return BigNumber.from(weiString);
+};
+
+const gWei = (amount) => {
+    const weiString = BigNumber.from("1000000000").mul(amount);
```

Comment:
@bejaxer we can use parseUnits here as well right? it is kinda hard to read

Reply:
> @bejaxer we can use parseUnits here as well right? it is kinda hard to read

BigNumber is useful, I think

Reply:
but you are using `parseUnits` in other utils methods, you can use BigNumber in L16 but in L15 can we use parseUnits?